staticjinja/staticjinja.py

```diff
@@ -334,11 +359,11 @@ def make_site(searchpath="templates",
         should store rendered files in. Defaults to ``'.'``.
 
     :param contexts:
-        A list of *(regex, function)* pairs. The Site will invoke
-        *function* if *regex* matches the name of a template when rendering.
-        *function* must take either no arguments or a single
-        :class:`jinja2.Template` as an argument and return a dictionary
-        representing the context. Defaults to ``[]``.
+        A list of *(regex, context)* pairs. The Site will use *context* if
+        *regex* matches the name of a template when rendering.  *context* must
+        be either acts as a dictionary or be a function which must take either
+        no arguments or a single :class:`jinja2.Template` as an argument and
+        return a dictionary representing the context. Defaults to ``[]``.
```

Comment:
Need to remove "acts".

Unrelated, but we should also write "Template whose name matches *regex* will access to *context* when rendering." instead of "The Site will use *context* if *regex* matches the name of a template when rendering."